README.md

```diff
@@ -109,7 +109,9 @@ To deploy to [production](https://was-registrar-app.stanford.edu): `bundle exec
 
 ## Auditing
 To audit the WARCs that have been accessioned in SDR against the WARCs available from a WASAPI provider,
-use the audit rake task: `bin/rake audit['<collection druid>']`
+use an audit rake task: 
+* For a collection that is configured in WRA: `bin/rake audit_collection['<collection druid>']`
+* For a collection that is not configured in WRA: `bin/rake audit['<collection_druid>',<wasapi_collection_id>,<wasapi_account>,<embargo_months>]`
 
 For example: `RAILS_ENV=production bin/rake audit['druid:hw105qf0103']`
```

Comment:
In this example, `audit` should now be `audit_collection`. 

Could you add an example for the collection not configured in WRA?

Reply:
Thanks for the catch. Fixed and added additional example.